### PR TITLE
chore(deps): Update Some Packages

### PR DESCRIPTION
**Pull request by ghost**
Update Some Packages

### CODE REVIEW

**Comment by MoustacheOff** [comment]
@discordjs/voice v13.12 currently causes an issue with encryption while connecting to a voice channel.

File: package.json [hunk 1]
```diff
@@ -53,35 +53,37 @@
 		"printWidth": 120
 	},
 	"dependencies": {
-		"@discordjs/voice": "^0.11.0",
+		"@discordjs/voice": "^0.13.0",
```
Comment (mzrtamp) [inline package.json:56]: Don't update this

File: package.json [hunk 2]
```diff
@@ -53,35 +53,37 @@
...
 		"@swc/cli": "0.1.57",
-		"@swc/core": "1.3.11",
+		"@swc/core": "1.3.14",
 		"date-fns": "2.29.3",
 		"discord.js": "13.12.0",
-		"dotenv": "16.0.3",
+		"dotenv": "16.0.3",
```
Comment (mzrtamp) [inline package.json:61]: Delete this space

File: package.json [hunk 3]
```diff
@@ -53,35 +53,37 @@
...
-		"@types/node": "18.11.8",
+		"@types/node": "18.11.9",
 		"@typescript-eslint/eslint-plugin": "5.42.0",
 		"@typescript-eslint/parser": "5.42.0",
 		"eslint": "8.27.0",
 		"eslint-config-prettier": "^8.5.0",
 		"eslint-plugin-prettier": "^4.2.1",
 		"prettier": "^2.7.1",
-		"rimraf": "^3.0.2",
+		"rimraf": "^3.0.2",
```
Comment (mzrtamp) [inline package.json:81]: Delete this space

File: package.json [hunk 4]
```diff
@@ -53,35 +53,37 @@
...
-		"rimraf": "^3.0.2",
+		"rimraf": "^3.0.2",		
 		"typescript": "4.8.4"
 	},
-	"optionalDependencies": {
-		"play-dl": "^1.9.6"
+	"optionalDependencies": {		
+		"ffmpeg-static": "5.1.0",
+		"play-dl": "^1.9.6",
+		"sodium-native": "3.4.1"
```
Comment (mzrtamp) [inline package.json:87]: Whats this

**Review by mzrtamp — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (Mednoob) [inline package.json:87]: `sodium-native` is not needed since we already have `tweetnacl` as prod deps

**Review by Mednoob — CHANGES_REQUESTED** [review]

**Pull request closed without merging**